### PR TITLE
DM-7464 Transfer files from firefly repo

### PR DESCRIPTION
**Pull request by stargaser**
The commits are a simple copy from the firefly repository, src/firefly/python/display directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+fc = FireflyClient('localhost:8080', 'mychannel')
+```
+
+A FITS image may be uploaded and displayed.
+
+```
+fval = fc.upload_file('image.fits')
+fc.show_fits(fval, 'myimage')
+
+
```
Comment (robyww) [inline README.md]: You might want to have some sort of reference back to the firefly repository.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+from setuptools import setup
+
+setup(
+    name='firefly_client',
+    version='1.0.0.b2',
+    description='Python API for Firefly',
+    author='IPAC LSST SUIT',
+    license='BSD',
+    url='http://github.com/Caltech-IPAC/firefly',
```
Comment (loitly) [inline setup.py]: is this url still valid?

**Review by loitly — APPROVED** [review]
Everything else looks fine.

File: README.md (hunk 1 above)
Comment (stargaser) [reply README.md]: I agree, a reference here would be good. Maybe also good to point to the release area for those wanting the standalone server.

File: setup.py (hunk 2 above)
Comment (stargaser) [reply setup.py]: Good catch. Looking at the pip packaging instructions, this should point to the firefly_client repository.

**Pull request merged**